gap/cmat.gi

```diff
@@ -46,8 +46,8 @@ InstallMethod( ConstructingFilter, "for a cmat",
 
 InstallMethod( NewMatrix, 
   "for IsCMatRep, a finite field, an integer, and finite field data",
-  [ IsCMatRep, IsField and IsFinite, IsInt, IsList ],
-  function( filt, f, rl, l )
+  [ IsCMatRep, IsField and IsFinite, IsList, IsInt ],
+  function( filt, f, l, rl )
```

Comment:
This should use `InstallOtherMethod` for now, so that it works now already. Moreover, we should retain the old method installation, so that we can make a release of `cvec` now, before the corresponding changes are in GAP.

So e.g. something like this:
```gap
BindGlobal( "NEW_CMAT_HELPER", function( filt, f, l, rl )
   ....
```
and then afterwards:
```gap
# for GAP <= 4.11
InstallOtherMethod( NewMatrix, 
   "for IsCMatRep, a finite field, an integer, and finite field data",
   [ IsCMatRep, IsField and IsFinite, IsInt, IsList ],
   { filt, f, rl, l } -> NEW_CMAT_HELPER( filt, f, l, rl ) );

# for GAP >= 4.12
InstallOtherMethod( NewMatrix, 
   "for IsCMatRep, a finite field, finite field data, and an integer",
   [ IsCMatRep, IsField and IsFinite, IsList, IsInt ],
   NEW_CMAT_HELPER );
```